data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -216,7 +227,8 @@ then
 		--asset-output-dir=/assets/kube-controller-manager-bootstrap \
 		--config-output-file=/assets/kube-controller-manager-bootstrap/config \
 		--cpc-config-output-file=/assets/kube-controller-manager-bootstrap/cpc-config \
-		--cluster-config-file=/assets/manifests/cluster-network-02-config.yml
+		--cluster-config-file=/assets/manifests/cluster-network-02-config.yml \
+    "${ADDITIONAL_FLAGS[@]}"
```

Comment:
indent seems to be off here too

Reply:
This is a backport, off indent is off here, it's also off in master. I wouldn't fix it.

Reply:
hm, ok.